.github/workflows/image-arm-pr.yaml

```diff
@@ -0,0 +1,25 @@
+name: 'Push latest ARM images'
+
+on:
+  pull_request:
+    paths:
+      - '**'
+
+concurrency:
+  group: ci-arm-${{ github.head_ref || github.ref }}-${{ github.repository }}
+  cancel-in-progress: true
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  docker:
+    uses: kairos-io/kairos/.github/workflows/reusable-docker-arm-build.yaml@1556-pr-one-flavor
```

Comment:
I'll change these to `master` right before merging. Right now if I do, it won't find the definition of that workflow.